### ISSUE TITLE
MacOS: cut: illegal option -- -

### ISSUE THREAD

**Issue by Terraro53** [pystardust/ani-cli#983]
**Metadata (please complete the following information)**
Version: 4.0.0
OS: macOS 13.2 22D49 arm64
Shell: zsh

**Describe the bug**
Upon typing in the name of anime, the program shows empty fzf list. When exited there is an error regarding cut.

**Steps To Reproduce**

1. Run `ani-cli flcl`
2. Empty fzf list is shown
3. Upon exit following error is shown 

> Checking dependencies...
cut: illegal option -- -
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-w | -d delim] [file ...]

**Expected behavior**
Script should work normally

**Additional context**
The error may have to do with version of cut on MacOS being diffrent from Linux, as the script works normally on linux.

**Comment by JorgePAJ** [comment]
Im having the same issue on mbp m1

**Comment by 71zenith** [comment]
can u check on the `bump` branch?

**Comment by JorgePAJ** [comment]
how do i install that branch?

**Comment by 71zenith** [comment]
```git clone "https://github.com/pystardust/ani-cli.git" -b bump && cd ./ani-cli
cp ./ani-cli "$(brew --prefix)"/bin 
cd .. && rm -rf ./ani-cli```

**Comment by Terraro53** [comment]
it doesnt fix it. out of curiosity i tested it on FreeBSD and got the same error. Since MacOS uses freeBSD utils i figured it would

**Comment by 71zenith** [comment]
Thanks for the info. I'll look into it

**Comment by Terraro53** [comment]
I did some digging. The issue occured on this commit 01a545ff80e235076bb60716052a6ca860158b7d. The commit before it works

**Comment by DanSM-5** [comment]
I tested on macos and looks like the cut version does not support the option `--output-delimiter` that gnu cut has producing the error message in the description: 
```
cut: illegal option -- -
usage: cut -b list [-n] [file ...]
cut -c list [file ...]
cut -f list [-s] [-w | -d delim] [file ...]
```


> I did some digging. The issue occured on this commit [01a545f](https://github.com/pystardust/ani-cli/commit/01a545ff80e235076bb60716052a6ca860158b7d). The commit before it works

I can see that the commit introduces this option in line 25.

**Comment by Tarzerk** [comment]
<img width="803" alt="image" src="https://user-images.githubusercontent.com/52636356/214471088-e0f340c1-e612-44d0-b416-0f2b2f5cea76.png">
<img width="910" alt="image" src="https://user-images.githubusercontent.com/52636356/214471123-c083344e-31e9-459b-a55b-c2bd8671c511.png">
I think I am having the same issue when I run the program. This happens

**Comment by 71zenith** [comment]
Pls try the `bump` repo again. If issue persists, do tell

**Comment by Terraro53** [comment]
issue is fixed for me

**Comment by 71zenith** [comment]
fixed in #993. closing issue
